problem3.swift

```diff
@@ -1,3 +1,25 @@
 // See problem3.README
 
+import Foundation
+
 let n = Int(readLine()!)!
+var i : Int = 2
+var prime = "true"
```

Comment:
`true` and `false` are actually just literals on their own, just like `1`, `2`, ..., `9`, & `0` are, except they are called Booleans (Bool for short). So, instead of writing
```var prime = "true"```
You could write
```var prime = true```
This will lead to way less possible mistakes as Booleans only have two possible values, `true` & `false`, while Strings can have any number of values.